### ISSUE TITLE
users cannot update profile pictures with verified emails

### ISSUE THREAD

**Issue by divine-comedian** [Giveth/giveth-dapps-v2#4906]
testing on production 

**users with verified emails cannot update their profile picture!**

when going to my account I can successfully add and verify my email address and save those changes. However when I go to update my profile picture I get an error saying Email is not verified, when it has already been verified in a previous step. 

I can however manage my projects, update information and save it with no issues

https://github.com/user-attachments/assets/47c1e341-f138-46ad-8417-f5ce0b9030e1

## AC 

- [x] User can verify their email on the app
- [x] users can set a PFP for their profile choosing a picture from their computer 
- [x] user can set PFP by choosing from eligible NFT in their wallet
- [x] user can create new profile with new email address and set profile picture

**Comment by divine-comedian** [comment]
@maryjaf this should be ready to test on staging and production

**Comment by MoeNick** [comment]
@divine-comedian can yp unblock @maryjaf ?

**Comment by divine-comedian** [comment]
@maryjaf 

I tested this on production and was successfully able to set a new Givers NFT as my PFP, I will call this passed. 

NFT page on production is working correctly, let's follow up why it isn't working on staging in another issue since it is lower priority.

**Comment by divine-comedian** [comment]
@maryjaf 

I took a closer look - the test NFTs we launched in 2023 on GOERLI - that testnet chain has been totally deprecated so likely there are no RPCs left to serve the data from the test NFT contract. 

We'll have to accept for now this feature won't work on staging.
